src/sentry/static/sentry/app/views/settings/components/settingsPageHeader.tsx

```diff
@@ -55,22 +71,15 @@ type TitleProps = {
 
 const TitleAndActions = styled('div')`
   display: flex;
-  align-items: center;
+  align-items: flex-start;
 `;
 const TitleWrapper = styled('div')`
   flex: 1;
 `;
 
 const Title = styled('div')<TitleProps & React.HTMLProps<HTMLDivElement>>`
-  ${p =>
-    !p.styled &&
-    `
-    font-size: 20px;
-    font-weight: bold;`};
-  margin: ${p =>
-    p.tabs
-      ? `${space(4)} ${space(2)} ${space(2)} 0`
-      : `${space(4)} ${space(2)} ${space(4)} 0`};
+  ${p => !p.styled && `font-size: 20px; font-weight: bold;`};
```

Comment:
```suggestion
  ${p => !p.styled && `font-size: 20px; font-weight: 600;`};
```
should we use a font-size from the theme?